src/pushy_messaging.erl

```diff
@@ -105,13 +105,25 @@ parse_message(Address, Header, Body, KeyFetch) ->
 build_message_record(_Address, Header, _Body) when size(Header) > ?MAX_HEADER_SIZE ->
     lager:error("Message rejected because header is too big ~p > ~p", [size(Header), ?MAX_HEADER_SIZE]),
     #pushy_message{validated = header_too_big};
-build_message_record(_Address, _Header, Body) when size(Body) > ?MAX_BODY_SIZE ->
-    lager:error("Message rejected because body is too big ~p > ~p", [size(Body), ?MAX_BODY_SIZE]),
-    #pushy_message{validated = body_too_big};
 build_message_record(Address, Header, Body) when is_binary(Header), is_binary(Body) ->
     Id = make_ref(),
+    MaxBodySize = envy:get(pushy, max_body_size, ?MAX_BODY_SIZE, integer),
```

Comment:
We're calculating size twice - `len_h(Body)` - let's capture that and use it for log + case.